spring-integration-jms/src/test/java/org/springframework/integration/jms/JmsOutboundGatewayTests.java

```diff
@@ -154,4 +181,104 @@ public Message answer(InvocationOnMock invocation) throws Throwable {
 			gateway.stop();
 		}
 	}
+
+	@Test
+	public void testConnectionBreakOnReplyMessageIdCorrelation() throws Exception {
+		CachingConnectionFactory connectionFactory1 = new CachingConnectionFactory(
+				new ActiveMQConnectionFactory("tcp://10.0.0.3:61616"));
```

Comment:
Looks like I missed adding my local change to change it back; re-pushed - I'll look at your other suggestions later.